src/NLog/Internal/XmlHelper.cs

```diff
@@ -58,9 +59,44 @@ public static class XmlHelper
         /// </summary>
         private static string RemoveInvalidXmlChars(string text)
         {
-            return String.IsNullOrEmpty(text) ? "" : InvalidXmlChars.Replace(text, "");
+            if (string.IsNullOrEmpty(text))
+                return string.Empty;
+
+#if NET3_5
+            return InvalidXmlChars.Replace(text, "");
+#else
+            for (int i = 0; i < text.Length; ++i)
+            {
+                char ch = text[i];
+                if (!XmlConvert.IsXmlChar(ch))
```

Comment:
@snakefoot I;m trying to include the changes of NLog 4.4 to NLog 5 (beta)

I seems that this method is not there in NETSTANDARD :(  (not until NETSTANDARD  2 - Q1 2017)

Any idea of an alternative?